org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/debug/AbstractRunHTMLDebugTab.java

```diff
@@ -156,13 +210,18 @@ public void initializeFrom(ILaunchConfiguration configuration) {
 	@Override
 	public void performApply(ILaunchConfigurationWorkingCopy configuration) {
 		String programPath = this.programPathText.getText();
-		configuration.setAttribute(AbstractHTMLDebugDelegate.PROGRAM, programPath);
+		if (programPathText.isEnabled()) {
+			configuration.setAttribute(AbstractHTMLDebugDelegate.PROGRAM, programPath);
```

Comment:
this should probably set the URL attribute to null

Reply:
I set it to an empty string `""` since setting it to `null` resulted in the compile error `The method setAttribute(String, String) is ambiguous for the type ILaunchConfigurationWorkingCopy`